src/sentry/static/sentry/app/views/eventsV2/table/cellAction.tsx

```diff
@@ -207,6 +359,21 @@ export default class CellAction extends React.Component<Props, State> {
     const {children} = this.props;
     const {isHovering} = this.state;
 
+    const {dataRow, column} = this.props;
+    const fieldAlias = getAggregateAlias(column.name);
+    const value = dataRow[fieldAlias];
+
+    // do not display per cell actions for count() and count_unique()
+    const shouldIgnoreColumn =
+      column.column.kind === 'function' &&
+      (column.column.function[0] === 'count' ||
+        column.column.function[0] === 'count_unique');
+
+    if (value === null || value === undefined || shouldIgnoreColumn) {
```

Comment:
I think we have a utility called `defined` for checking null and undefined at the same time.
https://github.com/getsentry/sentry/blob/8bad5792749fdae9fc106a245b14a7cf560098df/src/sentry/static/sentry/app/utils.tsx#L115-L117

Reply:
done in https://github.com/getsentry/sentry/pull/19230/commits/d481b481a3fc0f24bcdaa2ace854ba9764901574